tests/Unit/PointwiseFunctions/GeneralRelativity/Test_SpacetimeDerivativeOfGothG.cpp

```diff
@@ -0,0 +1,43 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Framework/CheckWithRandomValues.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Helpers/DataStructures/DataBox/TestHelpers.hpp"
+#include "PointwiseFunctions/GeneralRelativity/SpacetimeDerivativeOfGothG.hpp"
+
+namespace gr {
+
+template <typename DataType, size_t SpatialDim>
+void test_compute_spacetime_deriv_of_goth_g(const DataType& used_for_size) {
+  pypp::check_with_random_values<1>(
```

Comment:
Also test by making up values for the metric (doesn't have to be meaningful, just some smooth functions) and taking a numerical derivative.

Reply:
After further discussion between us, we decided that this one would not be worth the effort of pursuing.